source/documentors/references/doc_english_writing.rst

```diff
@@ -0,0 +1,220 @@
+Guidelines for Writing Global English
+#####################################
+
+
+.. contents::
+  :local:
+  :depth: 1
+
+***************
+Global Examples
+***************
+
+In addition to writing so that the text can be easily translated, remember
+that the Open edX audience is global, and make sure this is reflected in the examples
+that we use to illustrate concepts in the documentation. Use situations that
+people can relate to regardless of their culture, and use example names and
+email addresses that represent diverse ethnicities.
+
+
+***************
+Short Sentences
+***************
+
+Write short sentences when possible. Long sentences can include problems...
+
+* complex grammar
+* complex use of tenses
+* ambiguous pronouns
+* long clauses
+* complex modifier strings
+
+Try to limit sentences to 20-30 words. Rule of thumb: if you run out of breath
+when you read a sentence aloud, it is too long.
+
+However!
+
+* Do not damage clarity.
+* Do not sacrifice the syntactic cues provided by definite and indefinite articles.
+* Do not omit important information. Be explicit.
+
+Example:
+
+"Block open port on catheter fitting" could mean completely different things
+once the definite article is inserted. Compare "Block the open port on the
+catheter fitting." to "Block open the port on the catheter fitting.
+
+
+******************************
+Active Voice and Active Verbs
+******************************
+
+Use the active voice.
+
+Use active verbs. Avoid nominalizations (nouns made from verbs or adjectives) as they disguise the actors.
+
+  Example nominalizations:
+
+  utilization, operation, facilitation, activation, completion, reaction,
+  improvement, movement, discovery, difficulty
+
+Here's an example of a sentence drafted in the passive voice, and how it was rewritten in the active voice.
+
+  Passive: This method is typically applied to free text fields, including
+  discussion posts and wiki articles.
+
+  Active: Open edX typically applies this method to free text fields, including
```

Comment:
"Open edX" should not be used as a noun.

```suggestion
  Active: The Open edX platform typically applies this method to free text fields, including
```

Although, I find this example more confusing than helpful.

Reply:
I will review these suggestions and make the proper changes here and in the following documents. :)
I will include a note in this document and the style guide to ensure that the name "Open edX" is always accompanied by an article and is not used as a noun.